admin/admin-filters-term.php

```diff
@@ -634,40 +634,48 @@ public function get_inserted_term_language( $lang ) {
 			$lang_slug = sanitize_key( $_POST['term_lang_choice'] ); // phpcs:ignore WordPress.Security.NonceVerification
 			if ( ! empty( $lang_slug ) ) {
 				$lang = $this->model->get_language( $lang_slug );
+				return $lang instanceof PLL_Language ? $lang : null;
 			}
```

Comment:
If we want to reproduce the exact same behavior, I think we should do like this, here and in the other cases:

```suggestion
			if ( empty( $lang_slug ) ) {
				return null;
			}
			$lang = $this->model->get_language( $lang_slug );
			return $lang instanceof PLL_Language ? $lang : null;
```

Reply:
Even better indeed.